### PR TITLE
Updated models

### PR DESCRIPTION
**Pull request by rdotjain**
The changes made in this commit are as follows -

- Created an Announcements model in the hostel app
- Added a foreign key to hostel in Complaints model
- Updated delivery field in Shop model from `NullBooleanField` to `BooleanField`

### CODE REVIEW

**Comment by aniketbiswas21** [comment]
@Saurav-Shrivastav @ramyak-mehra Please take a look!

File: hashx/hostel/models.py [hunk 1]
```diff
@@ -134,3 +136,14 @@ class Room(models.Model):
...
+
+class Announcement(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    title = models.CharField(max_length=200, validators=[validate_is_profane])
+    description = models.TextField(max_length=1000, validators=[validate_is_profane])
+    date_posted = models.DateTimeField(default=timezone.now)
+    date_updated = models.DateTimeField(default=timezone.now)
+    user = models.ForeignKey(User, related_name='announcement',
+                                related_query_name='announcement', on_delete=models.SET_NULL, null=True, blank=True)
+    hostel = models.ForeignKey(Hostel, related_name='announcement',
```
Comment (ramyak-mehra) [inline hashx/hostel/models.py]: make this many to many since multiple hostels can get the same announcement.

**Review by ramyak-mehra — CHANGES_REQUESTED** [review]

File: hashx/hostel/models.py (hunk 1 above)
Comment (rdotjain) [reply hashx/hostel/models.py]: Done 👍🏻

**Review by Saurav-Shrivastav — APPROVED** [review]

**Review by AroraShreshth — COMMENTED** [review]
Null Boolean field is necessary . Null for this whose status isn't confirmed wether they offer delivery.

**Comment by AroraShreshth** [comment]
Have these been added these features in design ?

**Review by ramyak-mehra — APPROVED** [review]

**Comment by aniketbiswas21** [comment]
@ramyak-mehra @Saurav-Shrivastav @AroraShreshth Should I merge this?

File: hashx/shop/models.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ class Shop(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     name = models.CharField(max_length=100)
     about = models.CharField(max_length=1000, null=True)
-    delivery = models.NullBooleanField()
+    delivery = models.BooleanField(null=True)
```
Comment (ramyak-mehra) [inline hashx/shop/models.py]: @rdotjain make this by default False

**Pull request merged**